src/main/java/io/cdap/plugin/sfmc/sink/MarketingCloudConf.java

```diff
@@ -111,51 +112,72 @@
   @Description("Whether to truncate text that is longer than the max length specified in the data extension column.")
   private Boolean truncateText;
 
-  String getReferenceName() {
+  public String getReferenceName() {
```

Comment:
Do all these methods need to be public or is it sufficient for them to be package private? Similar comment for other places where methods have been made public

Reply:
Yes, these methods needs to be public for testing purpose.

Reply:
Are they used in tests in other packages? If not, they can just be package private

Reply:
yes it is used for same package, made it package private.

Reply:
Done